source/dashboards/actions.markdown

```diff
@@ -63,9 +63,9 @@ tap_action:
       default: "false"
     pipeline_id:
       required: false
-      description: "Assist pipeline id to use when the `action` is defined as `assist`"
+      description: "Assist pipeline to use when the `action` is defined as `assist`. It can be either `last_used`, `preferred` or a pipeline id."
```

Comment:
```suggestion
      description: "Assist pipeline to use when the `action` is defined as `assist`. It can be either `last_used`, `preferred`, or a pipeline id."
```